### PR TITLE
Merge taipy-templates

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**

### CODE REVIEW

**Review by trgiangdo — COMMENTED** [review]
The linter action is failing because templates use different linting rules due to some files are duplicated for different template.

We probably need to modify the linter action for templates.

**Review by trgiangdo — COMMENTED** [review]
Again, if possible, I would like to have the test action also runs taipy-templates tests in this PR.

**Review by jrobinAV — DISMISSED** [review]

File: src/__init__.py
Comment (jrobinAV) [inline src/__init__.py:1]: THis file should be removed, right?

**Review by jrobinAV — APPROVED** [review]

File: src/__init__.py
Comment (joaoandre-avaiga) [reply src/__init__.py:1]: This was removed on core's PR.

**Pull request merged**